test/parallel/test-repl-dynamic-import.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const child = child_process.spawn(process.execPath, [
+  '--interactive',
+  '--expose-gc'
+], {
+  stdio: 'pipe'
+});
+child.stdin.write('\nimport("fs");\n_.then(gc);\n');
+// Wait for concurrent GC to finish
+setTimeout(() => {
+  child.stdin.write('\nimport("fs");\n');
+  child.stdin.write('\nprocess.exit(0);\n');
+}, 500);
```

Comment:
500 milliseconds is pretty long. A lower value should be enough.

```suggestion
}, 50);
```

Reply:
It should also be `common.platformTimeout(50)`